bam_plcmd.c

```diff
@@ -811,8 +813,11 @@ static int mpileup(mplp_conf_t *conf, int n, char **fn, char **fn_idx)
             last_tid = tid0;
             last_pos = beg0-1;
             mplp_get_ref(data[0], tid0, &ref, &ref_len);
+        } else if (last_tid < 0 && !one_seq && conf->all > 1) {
+            last_tid = 0; // --aa on a blank file
+            mplp_get_ref(data[0], tid0, &ref, &ref_len);
         }
-       while (last_tid >= 0 && last_tid < sam_hdr_nref(h)) {
+        while (last_tid >= 0 && last_tid < sam_hdr_nref(h)) {
```

Comment:
on run it was observed that the data from 1st reference sequence is reused for all subsequent ones.

Reply:
Fixed by moving the mplp_get_ref into the loop.  Thanks